src/plugins/abc-notation/abc-notation-editor.js

```diff
@@ -16,7 +16,7 @@ function AbcNotationEditor({ content, onContentChanged }) {
   };
 
   const changeContent = newContentValues => {
-    onContentChanged({ ...content, ...newContentValues });
+    onContentChanged({ ...content, ...newContentValues }, false);
```

Comment:
I added this value for the second ("isInvalid") argument so that we see it when we need to add validation in a plugin

Reply:
:+1: Good idea!